static/app/views/preprod/utils.tsx

```diff
@@ -8,17 +8,15 @@ export function formatBytes(bytes: number, isSi = false) {
   return parseFloat((bytes / Math.pow(k, i)).toFixed(2)) + ' ' + sizes[i];
 }
 
-export function getPlatformFromArtifactType(
-  artifactType: BuildDetailsArtifactType
-): Platform {
-  switch (artifactType) {
-    case BuildDetailsArtifactType.XCARCHIVE:
-      return 'ios';
-    case BuildDetailsArtifactType.AAB:
-    case BuildDetailsArtifactType.APK:
+export function getPlatformIconFromPlatform(platform: Platform): string {
+  switch (platform) {
+    case 'ios':
+    case 'macos':
+      return 'apple';
```

Comment:
A bit torn here, on one end I want our own representation of platform as we want to discriminate iOS vs macOS, but at the same time it would be nice to just support the platforms already supported in https://sentry.sentry.io/stories/foundations/icons. Happy to make whatever work here, more of an iOS question than Android

Reply:
yea mapping these both to apple for the icon sgtm, we still want our data to keep them separate otherwise tho